.github/workflows/ubuntu-ppa-release.yml

```diff
@@ -121,6 +121,9 @@ jobs:
           # set target_ppa environment variable
           export TARGET_PPA="${{ vars.TARGET_PPA }}"
 
+          # set SIGN_KEY environment variable
+          export SIGN_KEY="${{ vars.SIGN_KEY }}"
```

Comment:
I think `SIGN_KEY` is too generic of a name.  I think it should contain `PPA` somewhere.

Reply:
revised as `PPA_SIGN_KEY`